src/client/components/DiscountTag/DiscountTag.tsx

```diff
@@ -0,0 +1,48 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import { useRedeemedCampaignsQuery } from 'data/graphql'
+import { getDiscountText } from 'pages/OfferNew/Introduction/Sidebar/utils'
+import { OfferData } from 'pages/OfferNew/types'
+import { isBundle, isNorwegian } from 'pages/OfferNew/utils'
+import { useTextKeys } from 'utils/textKeys'
+import { Badge } from '../Badge/Badge'
+
+type DiscountTagProps = {
+  offerData: OfferData
+}
+
+const DiscountInfo = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  align-items: flex-start;
+
+  *:not(:last-child) {
```

Comment:
nit: I like to explicitly add:

```suggestion
  & > *:not(:last-child) {
```